packages/components/src/card/card-header/index.js

```diff
@@ -15,6 +15,9 @@ export const defaultProps = {
 	size: 'medium',
 };
 
+/**
+ * @param { import('../types').HeaderProps & { className: string } } props
```

Comment:
Seems like both `HeaderProps` and `FooterProps` could use the `className` prop. Maybe we should just add it to `HeaderProps` and `FooterProps` will inherit it?

Reply:
Added the className props in https://github.com/WordPress/gutenberg/pull/32561/commits/6bce1b3ade071ad943058711c22e512d93af98d9